backend_new/src/controllers/listing.controller.ts

```diff
@@ -168,6 +155,19 @@ export class ListingController {
     return await this.listingService.process();
   }
 
+  @Put('clearCSV')
+  @ApiOperation({
+    summary: 'Trigger the removal of CSVs job',
+    operationId: 'clearCSV',
+  })
+  @ApiOkResponse({ type: SuccessDTO })
+  @PermissionAction(permissionActions.submit)
+  @UseInterceptors(ActivityLogInterceptor)
+  @UseGuards(OptionalAuthGuard, AdminOrJurisdictionalAdminGuard)
+  async clearCSV(): Promise<SuccessDTO> {
```

Comment:
Can you walk me through why we shifted to a cron job vs the unlink?

Reply:
Yeah! So we caught this with e2e testing, but basically if several requests for exports came in at the same time the creation of and deleting of files was causing collisions

Moving this to a cronjob cuts down on the possibility for collisions

Reply:
@YazeedLoonat Sounds good! I imagine we may need to bring this over to applications?

Reply:
@YazeedLoonat Were there changes here made to generalize this for apps?